dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml

```diff
@@ -8,14 +8,21 @@
       - @workshop.sessions.each do |session|
         = session.start_date_with_start_and_end_times_us_format
         %br
-  %tr
-    %td.heading
-      Location:
-    %td
-      = @workshop.location_name
-      - if @workshop.location_address.present?
-        %br
-        = @workshop.location_address
+  - unless @workshop.location_name.blank?
+    %tr
+      %td.heading
+        - if @workshop.virtual?
+          Link to join:
+        - else
+          Location:
+      %td
+        - if @workshop.virtual?
+          = link_to @workshop.location_name, @workshop.location_name, target: "_blank"
```

Comment:
I know we have some thoughts in the team about opening links in new tabs, would be nice to update this to align with those:

https://github.com/code-dot-org/code-dot-org/pull/37791

Reply:
oh cool, i learned something! I'll update the other links too